README.md

```diff
@@ -146,14 +147,14 @@ A simple TCP reverse proxy with SSL termination on port 993 and proxy protocol t
 									"handler": "proxy",
 									"proxy_protocol": "v1",
 									"upstreams": [
-										{"dial": ["localhost:1143"]}
+										{"dial": ["localhost:143"]}
 									]
 								}
 							]
 						}
 					]
 				},
-				"imaps-example": {
+				"normal-imaps": {
```

Comment:
I think this should be `normal-imap` because `imaps` is `imap + s` i.e. "secure"

Reply:
done